src/PillowResize/PillowResize.cc

```diff
@@ -18,6 +18,10 @@
 #include <cstdint>
 #include <utility>
 
+#ifdef _WIN32
+#define _USE_MATH_DEFINES
```

Comment:
If you do this check here, then move the `include <cmath>` below. (otherwise is not clear why you need this).

Also maybe is worth repeathing the same stuff (ifdef + define + include) everywhere <cmath> is included/used